client/coral-admin/src/routes/Community/containers/Community.js

```diff
@@ -29,6 +30,19 @@ const mapStateToProps = (state) => ({
   community: state.community,
 });
 
+
+const withFlaggedUsernamesCount = withQuery(gql`
+    query TalkAdmin_FlaggedUsernamesCount {
```

Comment:
Instead of creating multiple queries, there should be only one at the top most component that needs query data and assemble the data requirements of other containers using fragments like a tree.